docs/DafnyRef/Types.md

```diff
@@ -1088,6 +1088,13 @@ with arrays where parallel assignment is needed:
   }
 ```
 
+To convert an array `x` to a sequence, use the `[..]` operator:
+```dafny
+  var intArr: array<int> := new int[] [1, 2, 3];
+  var intSeq: seq<int> := intArr[..];
+  assert intSeq == [1, 2, 3];
+```
```

Comment:
Very nice!  But, should it be in the "Iterating over collections" chapter?

Reply:
Can you either move it there https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#sec-array-types
or at least make a reference there that points to this?

Reply:
Yes, of course, moved the explanation and added a reference to it.